components/tidb_query/src/codec/collation/utf8mb4.rs

```diff
@@ -6,7 +6,7 @@ use std::str;
 
```

Comment:
Suggest adding some comments about the behavior of these collations.

Reply:
done